docs/section5/Rule5-7-3.md

```diff
@@ -24,4 +24,4 @@
     - Check if space is EXISTING: `if _space.status_type == EXISTING:`
     - **Rule Assertion** `CANNOT_BE_VERIFIED`
```

Comment:
So if the RDS can verify that B-RMR = U-RMR then this would fall under the category where U-RMR needs to be compared to design documents to verify that it reflects design